.github/workflows/release.yml

```
@@ -254,7 +254,9 @@ jobs:
 
             - name: Complete release
               if: "needs.prepare.outputs.release_finish_cmd != ''"
-              run: eval yarn hardhat --network ${{ needs.prepare.outputs.release_network }} ${{ needs.prepare.outputs.release_finish_cmd }} ${{ needs.prepare.outputs.release_version }} ${{ needs.prepare.outputs.release_title }} ${{ needs.prepare.outputs.release_description }}
+              shell: bash
+              run: |
+                yarn hardhat --network ${{ needs.prepare.outputs.release_network }} ${{ needs.prepare.outputs.release_finish_cmd }} ${{ needs.prepare.outputs.release_version }} "${{ needs.prepare.outputs.release_title }}" "${{ needs.prepare.outputs.release_description }}"
```

https://wiki.polygon.technology/docs/develop/hardhat/#deploying-on-polygon-network

```suggestion
                yarn hardhat --network polygon_${{ needs.prepare.outputs.release_network }} ${{ needs.prepare.outputs.release_finish_cmd }} ${{ needs.prepare.outputs.release_version }} "${{ needs.prepare.outputs.release_title }}" "${{ needs.prepare.outputs.release_description }}"
```